server/appointment/upload-documents/uploadDocuments.php

```diff
@@ -263,6 +263,39 @@ function doesClientInformationMatch($clientInformation, $firstName, $lastName, $
 	return $firstNameMatches && $lastNameMatches && $emailAddressMatches && $phoneNumberMatches;
 }
 
+// container/blob naming rules here https://docs.microsoft.com/en-us/rest/api/storageservices/naming-and-referencing-containers--blobs--and-metadata
+function getContainerName($siteId) {
+	$containerName = null;
+	if($siteId == 1) {
+		$containerName = "nebraska-east-union";
+	} else if ($siteId == 2) {
+		$containerName = "victor-e-anderson-library";
+	} else if ($siteId == 3) {
+		$containerName = "jackie-gaughan-multicultural-center";
+	} else if ($siteId == 4) {
+		$containerName = "international-student-scholar";
+	} else if ($siteId == 5) {
+		$containerName = "center-for-people-in-need";
+	} else if ($siteId == 6) {
+		$containerName = "loren-eiseley-library";
+	} else if ($siteId == 7) {
+		$containerName = "bennett-martin-library";
+	} else if ($siteId == 8) {
+		$containerName = "f-street-community-center";
+	} else if ($siteId == 9) {
+		$containerName = "community-hope-federal-credit";
+	} else if ($siteId == 10) {
+		$containerName = "southeast-community-college";
+	} else if ($siteId == 11) {
+		$containerName = "nebraska-union";
+	} else if ($siteId == 12) {
+		$containerName = "virtual-vita";
+	} else if ($siteId == 13) {
+		$containerName = "student-athlete-virtual-site";
+	}
+	return $containerName;
```

Comment:
While this probably won't happen--I think this last return should be something like:
```php
return "unknown-site-$siteId";
```

Or something, just so we have a safe fallback other than returning `null`.